code/renderers/svelte/template/cli/ts/Button.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  import './button.css';
+
+  /**
+   * Is this the principal call to action on the page?
+   */
+  export let primary = false;
+
+  /**
+   * What background color to use
+   */
+  export let backgroundColor: string | undefined = undefined;
+  /**
+   * How large should the button be?
+   */
+  export let size: 'small' | 'medium' | 'large' = 'medium';
+  /**
+   * Button contents
+   */
+  export let label = '';
```

Comment:
```suggestion
  export let label;
```

I recommend no default, so that you test that your types picks up that it's required.

Reply:
Good catch fixed!